ethereum-explorer/src/map_filter_transactions.rs

```diff
@@ -0,0 +1,77 @@
+use substreams_ethereum::pb::eth::v2::{Block, TransactionTraceStatus};
+use crate::pb::eth::transaction::v1::{Transaction, Transactions};
+use serde::Deserialize;
+use substreams::errors::Error;
+use crate::util;
+
+#[derive(Deserialize)]
+struct TransactionFilterParams {
+    hash: Option<String>,
+    to: Option<String>,
+    from: Option<String>
+}
+
+#[substreams::handlers::map]
+pub fn map_filter_transactions(params: String, blk: Block) -> Result<Transactions, Vec<substreams::errors::Error>> {
```

Comment:
Some of my previous comments apply to here, mostly those around Rust. I think it will be more efficient if we review "live" by screen sharing so we can interact together quicker and go deeper into whys.